src/main/java/frc/robot/commands/shooter/StowShooter.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.commands.shooter;
+
+import edu.wpi.first.units.Units;
+// done
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.RobotConfig.ShooterConfig;
+import frc.robot.subsystems.shooter.Shooter;
+
+public class StowShooter extends Command {
```

Comment:
Should this override an end() method for when this is stopped?

Reply:
no, it's a whileheld